zombienet/tests/0007-full_node_warp_sync.feature

```diff
@@ -0,0 +1,13 @@
+Description: Full node catching up using warp sync
+Network: ./0007-full_node_warp_sync.toml
+Creds: config
+
+
+alice: is up
```

Comment:
default timeout maybe not enough for CI kubes. For substrate we have 30s (refer to the thread in: https://github.com/paritytech/substrate/pull/12890#issuecomment-1344234195)